### ISSUE TITLE
Looking to try and use this lib to allow AutoHotkey interop

### ISSUE THREAD

**Issue by evilC** [TyrenDe/streamdeck-client-csharp#4]
Hi, I wrote a library for the SD called [AutoHotStreamDeck ](https://github.com/evilC/AutoHotStreamDeck) ("AHSD")  
The goal is to allow an AutoHotkey script to handle button events, and also allow the AHK script to change button graphics.  
This lib uses pure HID to communicate with the SD - an AHK script loads the AHSD C# DLL, and can make calls to the DLL to change button graphics, and pass a function pointer to the C# DLL which gets fired when the specified button changes state.  
The big fly in the ointment here is that the SD is completely unaware that AHSD is handling one of the buttons - ie the user must make sure that button is left blank in the SD software  
The main problem I see is a fundamental difference in architecture - with AHSD as it stands, one AHK script can handle multiple SD buttons, and I would not really want to lose that functionality.  
Here is an example:  https://cdn.discordapp.com/attachments/330423308103319562/475730772586659870/2018-08-05_19_21_53.gif  
TLDR, I would like to allow one AHK script to be able to control imagery and react to button events of more than one button, whilst allowing other buttons on the same page to be controlled by the SD software  
Any advice gratefully received.

**Comment by TyrenDe** [comment]
The problem is, the Stream Deck SDK that they officially released doesn't allow you to interact with buttons that are NOT yours.  That is, when your plug-in connects, it only knows about the actions in which it needs to communicate about.

For example, I cannot make a full 15 button screensaver without my plug-ins actions being associated with all buttons.  Until they fix that, I'm not certain if you can do what you want.
